pingcap/dm
# syncer: add metric replicationLagGauge back

### What problem does this PR solve? 

in #1451 

*  Part 1 fix `replicationLagGauge`

### What is changed and how it works?

use  DM current  ts minus binlog header ts  to get the replication lag


### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test
   1. start any incremental task in any dm-worker
   2. check the workers metrics , you can find the `dm_syncer_replication_lag` back again

Code changes

 - Has exported function/method change
 - Add `GetServerUnixTS`

Side effects
 - Possible performance regression
1. need get the time offset between DM and  upstream db in background task    Regularly

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note
